.github/workflows/feature_deploy.yml

```diff
@@ -2,18 +2,18 @@ name: Feature environment deployment
 
 on:
```

Comment:
Can we remove this file please, as we should only have one `deployment.yml` to deploy to `feature` environment.

Reply:
Can we leave it in for now? There's quite a few things in here that mean it being separate is much easier to maintain - will list a few below:

1- we don't have a feature branch, we use feature branches to deploy to feature so there's no mapping between branch and environment
2- it uses dev env variables so there's a slight mismatch there that is hard-coded in that file
3- it wants to be deployed automatically on push to a branch rather than post QAT, and while we could add that to the deployment.yml file I'd feel safer if people are editing the feature_deploy.yml file for now so that they can't affect any dev/staging/prod related deployment logic
4- We don't deploy the functionapps right now on the feature environment but wouldn't want that logic in deployment.yml

If we do need to get rid of it I'd suggest we raise a ticket on the E2ETF board as it's not just a simple task